projects/manager/src/app/datasets/dataset-list/dataset-list.component.html

```diff
@@ -1,20 +1,33 @@
 <div class="container">
-    <div>
-        <mat-form-field appearance="fill">
-            <mat-label>Filter by name</mat-label>
-            <input matInput type="text" (input)="onSearchChange($event)" />
-        </mat-form-field>
-    </div>
-    <cdk-virtual-scroll-viewport class="bottom" [itemSize]="itemSizePx" (scrolledIndexChange)="onScrolledIndexChange($event)">
+    @if (addedDataset; as item) {
+        <button mat-stroked-button class="browseButton" (click)="backToAllDatasets()">
+            <mat-icon>chevron_left</mat-icon>Back to all datasets
+        </button>
         <mat-nav-list>
-            <ng-container *cdkVirtualFor="let item of source; trackBy: trackById">
-                <mat-list-item [class.selected]="(selectedDataset$ | async) === item" (click)="select(item)">
-                    <h4 matListItemTitle>{{ item.displayName }} [{{ item.name }}]</h4>
-                    <p matListItemLine>{{ item.description }}</p>
-                </mat-list-item>
-            </ng-container>
-        </mat-nav-list>
+            <mat-list-item class="selected">
+                <h4 matListItemTitle>{{ item.displayName }} [{{ item.name }}]</h4>
+                <p matListItemLine>{{ item.description }}</p>
+            </mat-list-item></mat-nav-list
+        >
+    } @else {
+        <div class="flex-container">
+            <mat-form-field appearance="fill">
+                <mat-label>Filter by name</mat-label>
+                <input matInput type="text" [(ngModel)]="searchName" (input)="onSearchChange($event)" />
+            </mat-form-field>
+            <button mat-raised-button color="primary" class="add-dataset" (click)="addDataset()">Add new dataset</button>
```

Comment:
![image](https://github.com/geo-engine/geoengine-ui/assets/4947061/8780cdcf-f043-4adf-bda4-9931b2b93f04)

This looks super clumsy. Could this be more elegant? Maybe make it not as high as the form input and not so stuffed at the border of the page.

Reply:
Can you make a concrete suggestion (code)? I don't think it makes sense that I change it until you like it.

Reply:
![image](https://github.com/geo-engine/geoengine-ui/assets/4947061/2d7400d6-c6bb-4c97-bb36-a87d09f1e4eb)

or just a fab with a plus

![image](https://github.com/geo-engine/geoengine-ui/assets/4947061/6b81348f-6fc5-4cd5-aa75-415488b6d9f9)

Reply:
c7516daf8afa2029792d65c321c3e0388f913c7b

Reply:
3d1fd27de8dac39946400104d4057f4e886cf524